homeassistant/components/netgear/sensor.py

```diff
@@ -339,11 +339,17 @@ def __init__(
     ) -> None:
         """Initialize a Netgear device."""
         super().__init__(coordinator, router, device)
+        self._device = device
         self._attribute = attribute
         self.entity_description = SENSOR_TYPES[attribute]
         self._attr_unique_id = f"{self._mac}-{attribute}"
         self._state = device.get(attribute)
 
+    @property
+    def available(self) -> bool:
+        """Return if entity is available."""
+        return self._device.get(self._attribute) is not None
```

Comment:
The superclass (the super super class) is a coordinator entity, so let's incorporate `super().available` as well (this makes sure that if we can't fetch data from the router for some reason, we definitely show as unavailable

Reply:
Done